toolium/driver_wrappers_pool.py

```diff
@@ -164,13 +167,18 @@ def close_drivers_and_download_videos(cls, name, test_passed=True, maintain_defa
                 # Stop driver
                 driver_wrapper.driver.quit()
                 # Download video if necessary
-                if (driver_wrapper.config.getboolean_optional('Server', 'video_enabled') or not test_passed) \
-                        and driver_wrapper.remote_node_video_enabled:
-                    driver_wrapper.utils.download_remote_video(driver_wrapper.remote_node, driver_wrapper.session_id,
-                                                               video_name.format(name, driver_index))
-            except Exception:
-                # Capture exceptions to avoid errors in teardown method due to session timeouts
-                pass
+                if not test_passed and driver_wrapper.config.getboolean_optional('Server', 'video_enabled', False):
```

Comment:
It should be 'or' instead of 'and', previous version download video when test fails or when video_enabled is true

Reply:
Fixed in 2bcebd5